styles.css

```diff
@@ -253,32 +329,174 @@
     height: 40px;
     background: #fff;
     border: 1px solid #8993a4;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #091e42;
   }
 
-  .second-tags {
+  .block-btn {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 10px;
+    width: 130px;
+    height: 50px;
+    background: #f55800;
+  }
+
+  .block-btn a {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #fff;
+    text-decoration: none;
+  }
```

Comment:
- [ ] As @roshan-bajgain suggested: **As per the project requirement, please implement the hovered, and `Pressed functionality. See the image below for reference.** We could use this [link](https://developer.mozilla.org/es/docs/Web/CSS/:hover#:~:text=La%20pseudo%2Dclase%20%3Ahover%20de,cursor%20(puntero%20del%20mouse).) to review about hover.

Reply:
you have been unfair to me as this is my last review and I will repeat the module. my mobile view was approved with the same media queries and buttons but you denied me from moving forward.

Reply:
Hi, @Lancelot-SO this is not my intention that you have to repeat the module remember that you can ask for three more reviews or a second opinion if you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 
I noticed that you are using the media queries at the beginning of your CSS file 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/206823344-ef0b895d-cc56-4cea-a2e6-9a64c60e6e89.png)

It is supposed that you should add it after the mobile styles if the mobile styles end in line 300 line 301 should be @media... and inside it should add the desktop styles.